force-app/main/default/classes/SOQL.cls

```diff
@@ -248,8 +289,11 @@ public with sharing class SOQL {
      * @return The Account Name of the Contact.
      */
     public static String getAccountNameForContact(String contactId) {
-        return null; // Replace null with the variable you used to store the result
-    }    
+
+        Contact contact = [SELECT AccountId FROM Contact WHERE Id = :contactId];
```

Comment:
This works well, but you can actually simplify to use a single query if you retrieve Account.Name from the Contact